resources/sdk/purecloudjava/templates/README.mustache

```diff
@@ -77,6 +77,42 @@ System.out.println("Authentication successful. Access token expires in " + authR
 
 ```
 
+See example on how to authenticate with an authorization code below. For more information see the article on [Code Authorization](https://developer.mypurecloud.com/api/rest/authorization/use-authorization-code.html)
```

Comment:
Shouldn't this statement should go a few lines down under the auth code heading?